mlir/test/lib/Tools/PDLL/TestPDLL.pdll

```diff
@@ -14,3 +14,9 @@ Pattern TestSimplePattern => replace op<test.simple> with op<test.success>;
 
 // Test the import of interfaces.
 Pattern TestInterface => replace _: CastOpInterface with op<test.success>;
+
```

Comment:
I think we should remove this test here. It's already covered in Parser/expr.pdll